packages/workflow/src/records/fhir.ts

```diff
@@ -298,6 +300,15 @@ export function createDocumentReferenceEntryForCertificate(
   }
 }
 
+function findPatientReferenceFromRelatedPerson(
+  record: RegisteredRecord | CertifiedRecord,
+  reference: ResourceIdentifier
+): ResourceIdentifier {
+  return (
+    findEntryFromBundle(record, reference)?.resource as Saved<RelatedPerson>
+  ).patient.reference
+}
```

Comment:
let's use `getResourceFromBundleById<RelatedPerson>` instead

Reply:
I don't think we can do that here because `getResourceFromBundleById` uses `getFromBundleById` which matches exactly the id whereas `findEntryFromBundle` uses the reference. We could handle the undefined case here though.